src/odemis/driver/actuator.py

```diff
@@ -940,6 +940,203 @@ def reference(self, axes):
+
+    def _updateReferenced(self):
+        """
+        update the referenced VA
+        """
+        # .referenced is copied to detect changes to it on next update
+        self.referenced._set_value(self._referenced.copy(), force_write=True)
+
+    @isasync
+    def moveRel(self, shift):
+        """
+        Move the rotation actuator by a defined shift.
+        shift dict(string-> float): name of the axis and shift
+        """
+        if not shift:
+            return model.InstantaneousFuture()
+        self._checkMoveRel(shift)
+        f = self._executor.submit(self._doMoveRel, shift)
+
+        return f
+
+    def _doMoveRel(self, shift):
+        """
+        shift dict(string-> float): name of the axis and shift
+        """
+        cur_pos = self._child.position.value[self._caxis] - self._offset
+        pos = cur_pos + shift[self._axis]
+        self._doMoveAbs({self._axis: pos})
+
+    @isasync
+    def moveAbs(self, pos):
+        """
+        Move the actuator to the defined position in m for each axis given.
+        pos dict(string-> float): name of the axis and position in m
+        """
+        self._move_num_total += 1
+        if self._move_num_total >= self._moves_before_ref:
+            self._child.reference({self._caxis}).result()
+            self._move_num_total = 0
+
+        if not pos:
+            return model.InstantaneousFuture()
+        self._checkMoveAbs(pos)
+        pos = self._applyInversion(pos)
+        f = self._executor.submit(self._doMoveAbs, pos)
+
+        return f
+
+    def _doMoveAbs(self, pos):
+        axis, distance = pos.items()[0]
+        logging.debug("Moving axis %s (-> %s) to %g", self._axis, self._caxis, distance)
+
+        try:
+            # While it's moving, don't listen to the intermediary positions,
+            # as it will not fit any known position, and be confusing.
+            self._child.position.unsubscribe(self._update_child_position)
+
+            move = {self._caxis: distance + self._offset}
+            self._child.moveAbs(move).result()
+
+        finally:
+            self._child.position.subscribe(self._update_child_position, init=True)
+
+    def _doReference(self, axes):
+        logging.debug("Referencing axis %s (-> %s)", self._axis, self._caxis)
+        f = self._child.reference({self._caxis})
+        f.result()
+
+    @isasync
+    def reference(self, axes):
+        if not axes:
+            return model.InstantaneousFuture()
+        self._checkReference(axes)
+
+        f = self._executor.submit(self._doReference, axes)
+        return f
+
+    reference.__doc__ = model.Actuator.reference.__doc__
+
+    def stop(self, axes=None):
+        """
+        stops the motion
+        axes (iterable or None): list of axes to stop, or None if all should be stopped
+        """
+        if axes is not None:
+            axes = set()
+            if self._axis in axes:
+                axes.add(self._caxis)
+
+        self._child.stop(axes=axes)
```

Comment:
As there is only one axis, you can just always call `self._child.stop({self._caxis})`